hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -167,6 +167,17 @@ def on_print_regs(sim: OTBNSim, args: List[str]) -> None:
         print(' w{:<2} = 0x{:064x}'.format(idx, value))
 
 
+def on_print_call_stack(sim: OTBNSim, args: List[str]) -> None:
+    '''Print call stack to stdout first element is top of stack'''
+    if len(args):
+        raise ValueError('print_call_stack expects zero arguments. Got {}.'
+                         .format(args))
+
+    print('PRINT_CALL_STACK')
+    for value in reversed(sim.state.gprs.peek_call_stack()):
```

Comment:
Reversing the call stack representation is likely to cause a debugging headache. Can you either reverse the representation everywhere, or change the output of this function?

Reply:
I'll remove the reverse here. I did this because I felt a stack is easier to read with the top element at the top (i.e. this reversed order). However I'm not sure anyone will ever actually look at this output directly, it's more for the ISS -> sim interface so can just dump it out as represented (i.e. top element at bottom of the output). @rswarbrick any thoughts?